algoliasearch/src/main/java/com/algolia/search/saas/Query.java

```diff
@@ -619,11 +619,11 @@ public int hashCode() {
 
     /**
      * Search for entries inside a given area defined by the two extreme points
-     * of a rectangle. At indexing, geoloc of an object should be set with
-     * _geoloc attribute containing lat and lng attributes (for example
-     * {"_geoloc":{"lat":48.853409, "lng":2.348800}})
+     * of a rectangle.
      * <p>
-     * You can use several bounding boxes (OR) by calling this method several times.
+     * At indexing, geoloc of an object should be set with
+     * {@code _geoloc} attribute containing lat and lng attributes (for example
+     * {@code {"_geoloc":{"lat":48.853409, "lng":2.348800}})}
```

Comment:
I would remove those comments. They are legacy from the API Client doc, but we should avoid duplicating this kind of information everywhere; instead, we should have a section (in the API Client doc) explaining how to specify geo information.